packages/aws-lambda/lambdas/wrapped-async/package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@instana/aws-lambda": "../../instana-aws-lambda.tgz",
     "@instana/serverless": "../../instana-serverless.tgz",
+    "@instana/core": "../../instana-core.tgz",
```

Comment:
There are 2 more lambda folders which make use of the generated tgz files. Should I add the core package there too? @basti1302

Reply:
@basti1302 Any opinion on that? :)

Packages are: just-creates-spans and timeout-detection

They don't seem to me like they would need to use a local core modification, but I am not sure

Reply:
That's a good question and I am not 100% sure, but I think the changes in `create-zips.sh` should be good enough. If we do this right, it should not matter which `@instana` dependencies are declared in the package.json files in the individual lambdas (or how they are declared), because that is completely managed by the script when deploying the Lambdas. WDYT?

Reply:
Yeah I think we don't need the core dependency here as e.g. aws-lambda uses it as sub dependency, which then should use the local core installation because of the `create-zips.sh` modification.
I'll re-test and remove that line here :)

Reply:
Works!Removed it